content/04.study.md

```diff
@@ -47,6 +47,32 @@ Deep learning applied to gene expression data is still in its infancy, but the f
+Furthermore, the latent space was visualized using t-SNE (t-distributed stochastic neighbor embedding) [@arxiv:1808.01359]. 
+Titus et. al. [@doi:10.5220/0006636401400145] adapted a VAE strategy developed by Way et. al. [@doi:10.1142/9789813235533_0008] to methylation data.
+The VAE was modified to perform dimensionality reduction on 300,000 PAM50-assigned CpG features to 100 latent features in 862 samples.
+The authors performed t-SNE visualization, clustering, and classified tumor subtypes from a Breast Cancer dataset from TCGA. 
+In an subsequent extension of this work [@doi:10.1101/433763v5], the authors constructed a 100-dimensional latent space of 100k CpG sites across around 1200 samples, and selected latent space dimensions that were the most highly associated with the differentiation between estrogen-response (ER) positive and negative tumor samples in breast cancer patients to determine the extent to which the latent space could predict responses to endocrine therapy. 
+Certain latent space dimensions differentiated tumors based on their ER status and provided biologically-plausible hypotheses, which suggests that VAE-derived models may have a place in summarizing DNA methylation profiles into composite features that can aid in predicting treatment response. 
+Another study explored the latent features of lung cancer methylation profiles that were extracted using variational autoencoders.
+After constructing a latent space representations of TCGA lung cancer samples, the authors used a logistic regression classifier on the latent dimentions to accturately classify cancer subtypes [@doi:10.1109/BIBM.2018.8621365]. 
+These studies, along with the growing body of work using VAEs and other latent representations of genomic and epigenomic data demonstrate a suite of tools to explore the unmeasured aspects of biology.
+These techniques are allowing for the discovery and analysis of latent biology that was previously under achieved using previously developed models.
+The power and robustness of unsupervised deep learning models comes from their ability to learn high-dimensional non-linear relationships among data. 
+
+Important applications in the future include predicting methylation and pathological states based on methylation profiles uncovered from datasets with more noise, such as solid tissue samples over blood samples. 
+A more complete understanding of the biological processes underlying cell types, transitions in cell dynamics, and subject phenotypes can be developed using unsupervised deep learning approaches such as variational autoencoders that leverage more of the measured data. 
```

Comment:
```suggestion
Unsupervised deep learning approaches such as variational autoencoders, which leverage measured points to produce a generative, low-dimensional representation, may provide a more complete understanding of the biological processes underlying cell types, transitions in cell dynamics, and subject phenotypes. 
```

Reply:
Looks like a lot changed, but I just reversed the sentence order to start with the methods